server/cmd/server/.env

```diff
@@ -8,12 +8,12 @@ WALLET_PATH="wallet.pem"
 WALLET_PASSWORD=""
 # MultiversX proxy (e.g.: https://testnet-gateway.multiversx.com)
 MULTIVERSX_PROXY="https://testnet-gateway.multiversx.com"
-# Multi Sig address on MultiversX to register the transactions
-MULTI_SIG_SC_ADDRESS="erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"
+# Header verifier address on MultiversX to register the transactions
+HEADER_VERIFIER_SC_ADDRESS="erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"
 # ESDT Safe address on MultiversX to execute the transactions
 ESDT_SAFE_SC_ADDRESS="erd1spyavw0956vq68xj8y4tenjpq2wd5a9p2c6j8gsz7ztyrnpxrruqzu66jx"
-# Max retries to wait in seconds for account nonce update after sending bridge txs
-MAX_RETRIES_SECONDS_WAIT_NONCE=60
+# Interval in milliseconds between sending bridge txs
```

Comment:
Miliseconds or seconds?

Reply:
This the interval between the txs if we would send them in bulk. We don't want to wait seconds between transfers